tools/wptrunner/wptrunner/executors/base.py

```diff
@@ -677,7 +677,7 @@ def run(self):
     def _run(self):
         try:
             self.result = True, self.func(self.session, self.path, self.timeout)
-        except (socket.timeout, IOError):
+        except (socket.timeout, OSError):
```

Comment:
Looks like this and all `socket.timeout` cases can be simplified since per https://docs.python.org/3/library/socket.html#socket.timeout it's a subclass of `OSError` now.

Reply:
I've filed https://github.com/asottile/pyupgrade/issues/471 about this.